provider/smsc/smsc.go

```diff
@@ -0,0 +1,63 @@
+package smsc
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+	"github.com/messagebird/sachet"
+)
+
+type SmscConfig struct {
+	Login    string `yaml:"login"`
+	Password string `yaml:"password"`
+	Sender   string `yaml:"sender"`
+}
+
+const SmscRequestTimeout = time.Second * 60
+
+type Smsc struct {
+	Login    string
+	Password string
+        Sender   string
+}
+
+func NewSmsc(config SmscConfig) *Smsc {
+	Smsc := &Smsc{Login: config.Login, Password: config.Password, Sender: config.Sender}
+	return Smsc
+}
+
+func (c *Smsc) Send(message sachet.Message) (err error) {
+	for _, number := range message.To {
+		err = c.SendOne(message, number)
+		if err != nil {
+			return fmt.Errorf("Failed to make API call to smsc:%s", err)
+		}
+	}
+	return
+}
+
+func (c *Smsc) SendOne(message sachet.Message, PhoneNumber string) (err error) {
+	fmt.Printf("ALERT : %s\n", message.Text)
+	encoded_message := url.QueryEscape(message.Text)
+	smsURL := fmt.Sprintf("https://smsc.ru/sys/send.php?login=%s&psw=%s&phones=%s&sender=%s&fmt=0&mes=%s", c.Login, c.Password, PhoneNumber, c.Sender, encoded_message)
```

Comment:
You should use here the message.From (https://github.com/messagebird/sachet/blob/e0e6edf4d019719ae58978adc31ddf13433520ec/sachet.go#L9) instead of `c.Sender`.

Reply:
you are right, fixed in 6d8dfa2